File: src/tasks/cli/command/reopen/ReopenArguments.java
package tasks.cli.command.reopen;

import omnia.data.structure.List;
import tasks.cli.arg.SimpleArguments;
import tasks.model.Task;

public final class ReopenArguments extends SimpleArguments {
  protected ReopenArguments(List<Task> tasks) {
    super(tasks);
  }
}

File: cli/src/me/strangepan/tasks/cli/feature/reopen/ReopenArguments.kt
package me.strangepan.tasks.cli.feature.reopen

import omnia.data.structure.List
import me.strangepan.tasks.cli.command.common.simple.SimpleArguments
import me.strangepan.tasks.engine.model.Task

/** Model for parsed Reopen command arguments.  */
class ReopenArguments internal constructor(tasks: List<Task>) : SimpleArguments(tasks)